js/components/adPanelTopBarItem.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * One item in ad panel component
+ */
+class AdPanelTopBarItem extends React.Component { // eslint-disable-line
```

Comment:
why do you ignore line instead of rewriting to functional component?

Reply:
I discovered that the HOC doesn't work as expected for pure components. Didn't spent enough of time to get a clear answer why.